samples/kubernetes/end2end/docs/08-alert.md

```diff
@@ -0,0 +1,81 @@
+## Firing Alerts
+In this step we show you how to fire alerts.
+
+### Configuring Alerts
+We do all the alerting related configurations in the customized [values.yaml](../prometheus/values.yaml) when installing the Prometheus chart.  
+
+First, configure alerting rules. For demonstration purpse, We define one [alerting rule](../prometheus/values.yaml#L45) named 'ClusterWarning' that will fire alerts if some WLS cluster has only one running server.
```

Comment:
Typo "purposes"
For this sample we define one [alerting rule](../prometheus/values.yaml#L45) named 'ClusterWarning' that will fire alerts when the WebLogic cluster goes down to having  a single server running.